pyracf/common/security_admin.py

```diff
@@ -54,6 +54,10 @@ class SecurityAdmin:
 
     _valid_segment_traits = {}
     _extracted_key_value_pair_segment_traits_map = {}
+    _racf_key_map = {}
+    _racf_message_key_map = {}
+    _racf_key_map = {}
+    _racf_message_key_map = {}
```

Comment:
Combine `_racf_key_map` and `_racf_message_key_map` into single structure? This reduces the number of arguments that need to get passed around by callers/users of our redaction function.

```python
_racf_trait_and_message_key_map = {}
```

```python
self._racf_trait_and_message_map = {
    "trait_map": {
        "audaltr": "audit",
        "audcntl": "audit",
        "audnone": "audit",
        "audread": "audit",
        "audupdt": "audit",
    },
    "message_map": {"uacc": "universal access"}
}
```

Reply:
Took a bit of refactoring, but that is all functional now. Good idea.